sdk/src/instruction_enable.rs

```diff
@@ -0,0 +1,7 @@
+// Is the instructions introspection feature enabled
```

Comment:
Would it make sense for this to be in `sysvar::instructions`?

Reply:
It would, but `sysvar::instructions` needs to be available from within feature = "program", but the genesis_config module is not compatible with feature = "Program"

Reply:
Ok, could featurize in `sysvar::instructions` rather than create a new file, don't feel strongly either way